pkg/provider/acceptance_connection_ssh_tunnel_test.go

```diff
@@ -64,10 +67,13 @@ func TestAccConnSshTunnel_update(t *testing.T) {
 				Config: testAccConnSshTunnelResource(roleName, connectionName, connection2Name, "mz_system"),
 			},
 			{
-				Config: testAccConnSshTunnelResource(roleName, newConnectionName, connection2Name, roleName),
+				Config: testAccConnSshTunnelResourceUpdated(roleName, newConnectionName, newHostName, newUser, newPort, connection2Name, roleName),
```

Comment:
Don't we still want to test the create as well?

Reply:
The create was still running in the previous step, but I've gone ahead and refined the test functions so that it's all in 1 function now rather than two separate ones. Thanks for that suggestion!